### PR TITLE
Various things...

### PR DESCRIPTION
**Pull request by michael1241**
I suck I know

### CODE REVIEW

**Comment by lakinwecker** [comment]
Thanks, I'll try and get to it this week. Feel free to pester me.

File: analyze.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def a1(working_set, report_name):
             fout.write(f'{player.username} ({result.min_rating} - {result.max_rating})\n')
             t_output(fout, result)
             fout.write(' '.join(result.game_list) + '\n')
-            fout.write('\n')
+            fout.write(str(len(result.game_list)) + ' games \n\n')
```
Comment (lakinwecker) [inline analyze.py:157]: You can use len() inside format strings, FYI

File: preprocess.py [hunk 2]
```diff
@@ -4,6 +4,10 @@
 import tqdm
 from multiprocessing import Pool, Process, Manager
 
+import sys
+
+sys.setrecursionlimit(10000)
```
Comment (lakinwecker) [inline preprocess.py:9]: I think this is a workaround for a deeper bug.

**Comment by lakinwecker** [comment]
I assume you've been using this since you made the PR?  Sorry for the ridiculous delay in reviewing it. It came in during a really busy time and then it just slipped my mind.

**Comment by michael1241** [comment]
Yep, it's been working fine as far as I can tell. Also I agree on the recursion limit being a workaround but it works and I have no idea where to even start debugging it.

**Pull request merged**